### PR TITLE
Parse only known args

### PR DESCRIPTION
**Pull request by agnieszkarybak**

### CODE REVIEW

File: bigflow/cli.py [hunk 1]
```diff
@@ -268,7 +268,7 @@ def _parse_args(project_name: Optional[str], args) -> Namespace:
 
     _create_codegen_parser(subparsers)
 
-    return parser.parse_args(args)
+    return parser.parse_known_args(args)[0]
```
Comment (grzegorzwitkowski) [inline bigflow/cli.py:271]: WDYT about making it more obvious that the goal here is to return only bigflow args without extra args?
```suggestion
    bigflow_only_args = parser.parse_known_args(args)[0]
    return bigflow_only_args
```

Comment (TSienki) [reply bigflow/cli.py:271]: Can we test it somehow?

Comment (agnieszkarybak) [reply bigflow/cli.py:271]: I'll add some tests once we decide we're good with the change at all :)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
         curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
         sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic test"
         sudo apt update
-        sudo apt install docker-ce
+        sudo apt install docker-ce containerd.io
```
Comment (artnowo-alle) [inline .github/workflows/ci.yml:38]: What is it needed for?

File: test/cli/test_cli.py [hunk 3]
```diff
@@ -727,6 +727,27 @@ def test_should_call_cli_build_commands_with_NOW_and_ALL(
...
+    @mock.patch('bigflow.build.operate.build_project')
+    @mock.patch('bigflow.build.spec.read_project_spec')
+    @freezegun.freeze_time("2001-02-03 15:00:00")
+    def test_should_call_cli_build_with_additional_params(
+        self,
+        read_project_spec: mock.Mock,
+        build_project_mock: mock.Mock,
+    ):
+        # when
+        cli(['build','--workflow', 'ALL', '--start-time', 'NOW', '--additional-param', 'param'])
```
Comment (grzegorzwitkowski) [inline test/cli/test_cli.py:739]: Both test look almost the same, despite different arguments being used. WDYT about using `@pytest.mark.parametrize`?

Comment (agnieszkarybak) [reply test/cli/test_cli.py]: I actually did that accidentally, look now

**Review by grzegorzwitkowski — APPROVED** [review]

**Pull request merged**